CONTRIBUTING.md

```diff
@@ -18,38 +55,23 @@ jpm run -b /usr/bin/firefox
 ```
 To know about add-on development in details do visit [MDN Add-on SDK](https://developer.mozilla.org/en-US/Add-ons/SDK)
 
-##Commit message
+firefox-search-extention is in its very early development stage, you can go over the issues on the 
+github issues page and send in a PR.
 
-firefox-search-extention is in its very early development stage, you can go over the issues on the github issues page and send in a PR.
+Note: The PR should be sent to the firefox branch and not the master branch
 
-Your commits in the PR should be of the form:
+your commits in the PR should be of the form:
```

Comment:
Everything else is good I added that format for the commit message as we do there in the chrome extension, which is what you removed.